Makefile

```diff
@@ -149,36 +117,6 @@ build-local-darwin-arm64: ## Build kusion tool chain for macOS arm64
 		-ldflags="-s -w" .
 
 build-local-darwin-arm64-all: build-local-darwin-arm64 ## Build kusion & kcl tool chain for macOS arm64
-	# Install kclvm darwin
-	go run ./scripts/install-kclvm \
-		--triple=Darwin-arm64 \
-		--mirrors=${KCLVM_URL_BASE_MIRRORS} \
-		--outdir=./_build/bundles/kusion-darwin-arm64/kclvm
-
-	# Build kcl-go
-	GOOS=darwin GOARCH=arm64 CGO_ENABLED=0 \
-		go build -o ${PWD}/_build/bundles/kusion-darwin-arm64/kclvm/bin/kcl-go \
-		./scripts/install-kcl-go
-
-	# Build kcl-openapi
-	cd ./scripts/install-kcl-openapi && GOOS=darwin GOARCH=arm64 CGO_ENABLED=0 \
```

Comment:
Maybe the kcl-openapi tool needs to be compiled separately here, because it is not included in kclvm-go.